consumer/src/events.rs

```diff
@@ -390,6 +398,7 @@ impl EventKind {
                     address: collection_mint.mint,
                 })
             },
+            Self::MintOpenDropBatched => unreachable!(),
```

Comment:
Why not instead of writing a different batch method you incorporate the logic here? Can have helper function for consolidating the logic.